bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ALP_linear_chiral/ZH_ALP_chiral/ZH_ALP_chiral_proc_card.dat

```diff
@@ -0,0 +1,5 @@
+import model ALP_chiral_UFO
+define l+ = e+ mu+
```

Comment:
What is the reason not to have Z -> tau tau, where tau can also give e, mu ?

Reply:
I do not know if I understand your first question correctly. As the author (Jorge Troconiz) said, the two models correspond to two EFT implementations: linear (simpler, contains ZZ and W+W- production) and chiral (more parameters, contains ZH production). And the models are uploaded to the CMS generator web repository. (https://cms-project-generators.web.cern.ch/cms-project-generators/)

For the Z -> tau tau, there is no objection to include taus as well.  I just committed a new version to add it.